packages/web-app-files/tests/unit/components/SideBar/PrivateLinkItem.spec.ts

```diff
@@ -11,13 +11,21 @@ describe('PrivateLinkItem', () => {
     expect(wrapper.html()).toMatchSnapshot()
   })
   it('upon clicking it should copy the private link to the clipboard button, render a success message and change icon for half a second', async () => {
-    jest.spyOn(window, 'prompt').mockImplementation()
+    Object.assign(window.navigator, {
+      clipboard: {
+        writeText: jest.fn().mockImplementation(() => Promise.resolve())
+      }
+    })
+
     const { wrapper } = getWrapper()
     const spyShowMessage = jest.spyOn(wrapper.vm, 'showMessage')
     expect(spyShowMessage).not.toHaveBeenCalled()
 
     await wrapper.trigger('click')
     expect(wrapper.html()).toMatchSnapshot()
+    expect(window.navigator.clipboard.writeText).toHaveBeenCalledWith(
+      wrapper.vm._provided.displayedItem.privateLink
+    )
```

Comment:
Please no private api. Can't you just hardcode the value?

Reply:
Yeah I was more concerned with whether I could rather than whether I should :D fixed it by extracting the mocked folder, thx